client/src/api/recipesApi.js

```diff
@@ -0,0 +1,17 @@
+import { extractData } from './helpers';
+import request from './request';
+
+const urls = {
+  root: '/recipes',
+  get browse() {
+    return this.root + '/browse';
```

Comment:
the two others are called `/list`, I don't see why this shouldn't be `/list` also

Reply:
the same goes for the function name